blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineStepImpl.java

```diff
@@ -241,6 +243,12 @@ private Object parseValue(InputStepExecution execution, JSONArray parameters, St
             }
             mapResult.put(name, convert(name, v));
         }
+        // If a destination value is specified, push the submitter to it.
+        String valueName = input.getSubmitterParameter();
+        if (valueName != null && !valueName.isEmpty()) {
```

Comment:
@vivek as @jglick pointed out, this comes from https://github.com/jenkinsci/pipeline-input-step-plugin/blob/1031ad8008f2b84e1ca331e3eff7839d97922169/src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepExecution.java how could we reduce that redundancy?